app/views/scans/_ineligible_exception.html.erb

```diff
@@ -0,0 +1,5 @@
+<strong class="h3">Scan-to-PDF not available.</strong>
+
+<p>
+This pilot service is currently available only to faculty, grad students, postdocs, academic staff, fellows, and visiting scholars. Your feedback will help us refine and expand the service!
+</p>
```

Comment:
This content is currently in an i18n string.  I think it was used somewhere early on in the project, but I think it has been removed (if it ever even made it into the UI). I think we should either set the flash message to the i18n string or kill it and define the message here (as you have done).

It seems like there is some logic around making the partial HTML safe for the flash message, and I wonder if the *_html i18n string would give us that for free.